backend/experiment/actions/wrappers.py

```diff
@@ -96,7 +96,8 @@ def song_sync(session, section, title, response_time=15, play_method='BUFFER'):
                         play_config={
             'ready_time': 0,
             'playhead': silence_time + (random.randint(100, 150) / 10 if not continuation_correctness else 0),
-            'show_animation': True
+            'show_animation': True,
+            'resume_play': True
```

Comment:
And why is it `True` here? :D

Reply:
We only set it to `True` in any view that is supposed to "resume play", which is the case in this particular stage of the Hooked experiment, where a player gets to hear a continuation after a silence time and has to indicate whether the track continued at the expected, or unexpected moment.